### PR TITLE
Add vee-validate

### PR DESCRIPTION
**Pull request by logaretm**
Adds vee-validate VueJS plugin update.json and info.ini

### CODE REVIEW

File: files/vee-validate/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "vee-validate",
+  "repo": "logaretm/vee-validate",
+  "files": {
+    "include": ["vee-validate.js", "vee-validate.min.js"]
```
Comment (megawac) [inline files/vee-validate/update.json]: you missed `basePath: "dist"` I think

Comment (logaretm) [inline files/vee-validate/update.json]: sorry about that, just added it.

**Comment by megawac** [comment]
sorry for late review, been busy

**Comment by logaretm** [comment]
Its okay, I can see you have enough on your plate over here :)

**Pull request merged**